packages/web-api/src/controllerDispatcher.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Context } from '@azure/functions';
+import { Container } from 'inversify';
+import { BaseTelemetryProperties } from 'logger';
+import { ProcessEntryPointBase } from 'service-library';
+import { ApiController } from './controllers/api-controller';
+import { Newable, webApiIocTypes } from './setupIoContainer';
+
+// tslint:disable: no-any no-unsafe-any
+
+export class ControllerDispatcher extends ProcessEntryPointBase {
+    private readonly controller: ApiController;
+
+    constructor(private readonly controllerType: Newable<ApiController>, context: Context, container: Container) {
+        super(container);
+        container.bind(webApiIocTypes.azureFunctionContext).toConstantValue(context);
```

Comment:
>  container.bind(webApiIocTypes.azureFunctionContext).toConstantValue(context); [](start = 6, length = 79)

how will this work in parallel processing? handling 2 requests at the same time? will it be working on different containers in that case? if not, then you will have conflict with the azurefunctioncontext

Reply:
this is single request processing